src/components/pagination/README.md

```diff
@@ -0,0 +1,29 @@
+# d2l-labs-pagination
+
+A component to indicate the existence of, and provide navigation for, multiple pages of content.
+
+## Usage
+
+```html
+<script type="module">
+    import '@brightspace-ui-labs/pagination/pagination.js';
+</script>
+<d2l-labs-pagination></d2l-labs-pagination>
+```
+
+## Properties
+
+| Property | Type | Description |
+|--|--|--|
+| `page-number` | Number, default: 1 | The current page number
```

Comment:
In the old repo, a couple of these were listed as required. Is that not stated in this repo, replaced by the default value, no longer true, or not here for some other reason?

Reply:
So what I've noticed is that in other component cases, we don't actually have the `required` part in favour of saying what the default is since in theory the component still works without giving the `page-number`, but if that's done, you just won't be able to do much with it. I can add back the `required` field though, for both this and for the max page number